test/borough/borough.repository.mock.ts

```diff
@@ -2,12 +2,17 @@ import {
   findManyRepoSchema,
   checkByIdRepoSchema,
   findCommunityDistrictsByBoroughIdRepoSchema,
+  findCapitalProjectsByBoroughIdCommunityDistrictIdRepoSchema,
 } from "src/borough/borough.repository.schema";
+// import { checkByCommunityDistrictIdRepoSchema, checkCommunityDistrictByIdMocks } from "src/community-district/community-district.repository.schema";
```

Comment:
Will remove this, clean up and squash once there's feedback about which wya to go for checking for community district ID

Reply:
Yeah, it seems to make more sense to have the "community district id check" be in the community district repository